requirements.txt

```diff
@@ -0,0 +1 @@
+pillow
```

Comment:
We already have this is `setup.py`.
`imgp` works with both `pillow-simd` and `pillow`.

Please remove this file.